test/module/irohad/consensus/yac/yac_test_util.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_YAC_TEST_UTIL_HPP
+#define IROHA_YAC_TEST_UTIL_HPP
+
+#include <gmock/gmock.h>
+
+#include "consensus/yac/vote_message.hpp"
+#include "consensus/yac/yac_hash_provider.hpp"
+
+#include "module/irohad/consensus/yac/mock_yac_crypto_provider.hpp"
+#include "module/shared_model/interface_mocks.hpp"
+
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+
+      inline std::shared_ptr<shared_model::interface::Peer> mk_peer(
```

Comment:
Is function named in such way for purpose? If not, it should be renamed to `mkPeer` or even just `makePeer`